spec/controllers/idv/sessions_controller_spec.rb

```diff
@@ -101,7 +101,7 @@
     end
 
     context 'with in person enrollment' do
-      let(:user) { build(:user, :with_pending_in_person_enrollment) }
+      let(:user) { create(:user, :with_pending_in_person_enrollment) }
```

Comment:
What's the benefit of using `create` instead of `build`?  (I'm not against it, just wondering about the motivation for the change.)

Reply:
It became necessary after switching to `association`. I'm not entirely clear the reason why, though it's likely that the previous `build` ended up doing a `create` anyways, due to the lifecycle `after :build` `create`'ing the associated `in_person_enrollment`. It may be that record relationships require database persistence rather than just local unsaved copies.

Reply:
Roger, thank you.